CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 0.6.0 - Bump plutus version to 1.7.0.0
```

Comment:
Should this be renamed after the commit that reverted back to 1.0

Reply:
Oh 1.7.0.0 is version of plutus-the-haskell-library, not plutus-the-language so it's fine